### DISCUSSION TITLE
flat config Phase3 list consider adding angular-eslint

### DISCUSSION THREAD

**Discussion by SimonFischer04** [eslint/eslint#17578] [General] [answered]
https://github.com/angular-eslint/angular-eslint is the monorepo for everything eslint related in angular.

I think this should be added to the compatibility testing list https://github.com/eslint/eslint/issues/13481 in Phase3 as well, as the @angular-eslint/eslint-plugin alone has ~850 000 downloads/week.

**Comment by nzakas** [comment, marked as answer]
Thanks for the suggestion. Our bandwidth for compatibility testing is limited, so we aren't adding any further projects right now. If there end up being issues, we'll happily help out when issues are filed.

**Comment by SimonFischer04** [comment]
Ok, understandable for a communuty project. Thanks
